Sources/UserDefault.swift

```diff
@@ -49,6 +49,10 @@ public struct UserDefault<T> where T: UserDefaultElement {
                 } else {
                     UserDefaults.standard.set(object, forKey: key)
                 }
+
+                // Post on the `NotificationCenter`
+                NotificationCenter.default.post(userDefault: self, object: self)
```

Comment:
Note it's the `UserDefault<T>` which is posted, rather than the wrapped `T`.

Reply:
What's the reasoning for this? :D My generics mind isn't in gear today :P

Reply:
Guess I thought it allows you to check both the `key` which it's saved under and the `wrappedValue` itself 😃

Reply:
Ahh yep of course! :D